src/uwtools/tests/fixtures/rocoto_invalid.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE workflow [
+  <!ENTITY ACCOUNT "myaccount">
+  <!ENTITY FOO "test.log">
+
+]>
+<workflow realtime="False" scheduler="slurm" >
+  <!-- cycldef tag has intentionally been removed to fail Rocoto schema validation
+  cycledef is required and defines the set of cycles to be run for the workflow
+  -->
+  <log>/some/path/to/&FOO;</log>
+  
+  <task name="hello" cycledefs="howdy" maxtries="2" >
```

Comment:
You might consider removing all task and metatask entries, too, just to keep content shorter.